pages/home/EmailSignupForm.jsx

```diff
@@ -20,10 +20,10 @@ var PledgeSignupForm = React.createClass({
       return;
     }
 
-    if (process.env.NODE_ENV !== 'production' && !process.env.PLEDGE_MAILINGLIST_URL) {
+    if (process.env.NODE_ENV !== 'production' && !process.env.NEWSLETTER_MAILINGLIST_URL) {
       e.preventDefault();
-      alert("PLEDGE_MAILINGLIST_URL is not defined. Simulating a successful pledge signup now.");
-      window.location = "?pledge=thanks";
+      alert("NEWSLETTER_MAILINGLIST_URL is not defined. Simulating a successful newsletter signup now.");
+      window.location = "?signup=thanks";
```

Comment:
cool, I talked to @alicoding and I will mimic how the Advocacy site is doing it: https://github.com/mozilla/advocacy.mozilla.org/blob/eaad169217ac0e734b78d644566e993bcb3ce79a/components/encrypt-signup.jsx#L23